SchibstedAccount/UI/IdentityUIRoute.swift

```diff
@@ -19,10 +19,16 @@ extension IdentityUI {
         /// Route to the next step after email validation occurred after signup.
         case validateAuthCode(code: String, shouldPersistUser: Bool)
 
+        /// Route to present updated terms and condition to an already logged-in user.
+        case presentUpdatedTerms
```

Comment:
Should this just be `presentTerms` maybe? There should be no difference from the SDK point of view of updated terms or terms. They are also pretty much the same data/screen in the backend. And this should ideally have an associated property of `User`.

I realize that our current design is not going to work with only passing User here, so it should probably be IdentityManager goes in here :(

Reply:
Ok for `presentTerms`.

Yeah, so I was passing the `IdentityManager` to the `presentRoute...` method instead. But maybe it can now go back to the initializer.